e2e/testdata/fn-render/path-index-duplicate/resources.yaml

```diff
@@ -0,0 +1,17 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+apiVersion: apps/v1 # Copyright 2021 Google LLC
```

Comment:
The comment `Copyright` is misplaced.

Reply:
ah my bad, will fix. must have forgotten to restore the resource after manual testing